src/views/VideoMania/index.jsx

```
@@ -19,27 +15,31 @@ import videoIcon from "../../assets/videoIcon.svg";
 import { AddVideoMania } from "../../services/videomania";
+import Player from "../../components/player";
 
 const VideoMania = () => {
   const [searchQuery, setSearchQuery] = useState("");
   const [currentPage, setCurrentPage] = useState(1);
   const [rowsPerPage, setRowsPerPage] = useState(10);
-  const [currentId, setCurrentId] = useState(null);
-  const [activeCategory, setActiveCategory] = useState(null);
+
   const [addModal, setAddModal] = useState(false);
   const [editModal, setEditModal] = useState(false);
```

ESLint

[error] 'editModal' is assigned a value but never used.

(no-unused-vars)